db/deploy/computed_columns/application_status.sql

```diff
@@ -14,6 +14,7 @@ grant execute on function ccbc_public.application_status to ccbc_auth_user;
 grant execute on function ccbc_public.application_status to ccbc_job_executor;
 grant execute on function ccbc_public.application_status to ccbc_analyst;
 grant execute on function ccbc_public.application_status to ccbc_admin;
+grant execute on function ccbc_public.application_status to public; -- needed for Metabase
```

Comment:
Please comment on that. I assume it is safe but please correct me if I'm wrong. Any alternative solutions are welcome.

Reply:
I'm tempted to say that it's safe as well, since this is neither volatile nor is it exposed as an endpoint in graphql for it to be abused. 

Alternatively, option 2 for this is to go right into the DB and grant execute function manually to the ccbc_readonly user. I'd rather have this saved in our codebase though. 

I'm okay with this though. Do we know why the application_status function works fine in test for metabase but not on prod? If we've already done option 2 for test may as well stay consistent

Reply:
I vote for option 2 :) . I will add commented out line here and update README just to keep track.